### PR TITLE
Edite README.md

### PR DESCRIPTION
**Pull request by hongqian-li**
Add Hongqian's part into the README.md

Please check and review my file, thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## Authors
+
+* Hongqian Li - Initial work - [HongqianLi](https://github.com/hongqian-li)
+* Richard Švajdlenka - Database Integration - [Richard](https://github.com/M3mberTM)
+* Julian Hack - System Testing - [JulianHack](https://github.com/Julian23000)
+* Attila Vegh - Backend Development - [AttilaVegh](https://github.com/Attila22000)
+
+
+### Hongqian Li
+Main responsibilities include ensuring the continuity and integration of content from the previous robotics project, facilitating team meetings, managing Jira and Confluence, and providing code and testing support to the team to complete basic initial work.
```
Comment (M3mberTM) [inline README.md:14]: This is very clearly written by ChatGPT. I understand you having hard time trying to write it in english but you should still try. It needs to be your words.

**Review by M3mberTM — CHANGES_REQUESTED** [review]
Fix the ChatGPT text

**Review by Julian23000 — CHANGES_REQUESTED** [review]
Change my obligation to Researcher and Debugger

File: README.md (hunk 1 above)
Comment (hongqian-li) [reply README.md:14]: Thanks for your feedback, I will do it by my own.

**Review by M3mberTM — APPROVED** [review]
Happy that you listened to my advice. This seems so much better. Keep up the work!

**Review by Julian23000 — APPROVED** [review]
Thank you for changing it!

**Pull request merged**